src/basic/terminal-util.c

```diff
@@ -1399,7 +1399,7 @@ bool getenv_terminal_is_dumb(void) {
 
         e = getenv("TERM");
         if (!e)
-                return true;
+                return false;
```

Comment:
Hmm, I doubt we should flip this. If there's no `$TERM`, it supposedly indicates a faulty terminal environment or no terminal shall be used by the process at all.

What's the exact rationale here?

Reply:
I figured it might help with the issue reported it in the linked PR.

Though maybe the second commit would be sufficient.